api/user/schema.py

```diff
@@ -242,7 +242,7 @@ def mutate(self, info, **kwargs):
         location_id = kwargs['location_id']
         query_user = User.get_query(info)
         user = query_user.filter(
-            UserModel.state == "active", UserModel.id == logged_in_user.id).first() # noqa
+            UserModel.state == "active", UserModel.id == logged_in_user.id).first()  # noqa
```

Comment:
@vic3king This check is not necessary as a user with a non-active account will not be able to use this mutation.

Reply:
True @rajeman . I'll update it.